src/test/ui/nll/user-annotations/constant-in-expr-inherent-2.rs

```diff
@@ -1,6 +1,10 @@
 // Test that we still check constants are well-formed, even when we there's no
 // type annotation to check.
 
+// revisions: base nll
+// ignore-compare-mode-nll
+//[nll] compile-flags: -Z borrowck=mir
```

Comment:
Why was this test made a revision, too? The output seems to be the same?

Reply:
Mistake, I think. Reverted this change.